opsdroid/connector/slack/__init__.py

```diff
@@ -193,3 +225,12 @@ def __init__(self, config, opsdroid=None):
             message = message.replace("<@{userid}>".format(userid=userid),
                                       user_info["name"])
         return message
+
+    async def check_for_files_access(self):
+        """Check whether bot token has access to Slack files."""
+        try:
+            await self.slacker.files.list()
+        except slacker.Error as error:
```

Comment:
In Python 3.5.7 this exception is not being caught. This is due to a bug that is not present in 3.6.8. I'm planning on putting this PR on hold until #953 has been discussed thoroughly.